dev/tox.ini

```diff
@@ -19,6 +19,6 @@ max-line-length=100
 exclude=python/pyspark/cloudpickle/*.py,shared.py,python/docs/source/conf.py,work/*/*.py,python/.eggs/*,dist/*,.git/*
 
 [flake8]
-select = E901,E999,F821,F822,F823,F401
-exclude = python/pyspark/cloudpickle/*.py,shared.py,python/docs/source/conf.py,work/*/*.py,python/.eggs/*,dist/*,.git/*
+select = E901,E999,F821,F822,F823,F401,F405
+exclude = python/pyspark/cloudpickle/*.py,shared.py,python/docs/source/conf.py,work/*/*.py,python/.eggs/*,dist/*,.git/*,dev/*
```

Comment:
Why did we exclude `dev/*`?

Reply:
It was generating a noise, I've added it now again.

Reply:
I've reverted the change and fixed the violations in `dev/*` as well